### PR TITLE
typo in config.yaml

### LINKED ISSUE #122: API ref: typo in config.yaml
See PR #117.

### PR DESCRIPTION
**Pull request by kbhawkey**
Change CSINode version from v1 to v1beta1.
Closes #122.

### CODE REVIEW

File: gen-apidocs/config/config.yaml [hunk 1]
```diff
@@ -82,7 +82,7 @@ resource_categories:
       version: v1beta1
       group: storage
     - name: CSINode
-      version: v1
+      version: v1beta1
```
Comment (tengqm) [inline gen-apidocs/config/config.yaml]: no. CSINode is v1  GA now.

Comment (kbhawkey) [reply gen-apidocs/config/config.yaml]: OK. The change was made after the 1.17 release?

Comment (tengqm) [reply gen-apidocs/config/config.yaml]: Just checked: https://github.com/kubernetes/kubernetes/commit/9268d4053928488ea58829484bd59537b89da205#diff-ccf355e02c6054937bad28a1d8d754b4

msau42 committed on Oct 19, 2019

It is part of 1.17 release.

**Comment by kbhawkey** [comment]
OK, thanks for checking.

**Pull request closed without merging**